pkg/util/cert/cert.go

```diff
@@ -53,11 +54,16 @@ func BootstrapCerts(kubeConfig *rest.Config, cfg config.Configuration) error {
 			BindAddress:    "0",
 			FilterProvider: filters.WithAuthenticationAndAuthorization,
```

Comment:
I'm wondering we should drop this in a follow up, this is not related to the bug. However, I think this was just copy-paste. We don't expose metrics in the bootstap manager (at least currently), so I think no need for the FilterProvider: WithAuthenticationAndAuthorization.